tests/run-make/asm-incr-cache/rmake.rs

```diff
@@ -0,0 +1,16 @@
+extern crate run_make_support;
+
+use run_make_support::{rustc, tmp_dir};
+
+fn main() {
+    let inc_dir = tmp_dir();
+
+    for _ in 0..=1 {
+        rustc()
+            .input("lib.rs")
+            .arg("--emit=obj,asm,dep-info,link,mir,llvm-ir,llvm-bc")
```

Comment:
nit: you could also use the helpers you introduced for this test

Reply:
Technically this gets added in a different pull request so I just add a separate commit in this one where helpers are added to update the test.